src/utils/project.js

```diff
@@ -8,11 +8,13 @@
 
 const fs = require('fs');
 const path = require('path');
-const GlobSync = require('glob').GlobSync;
+const fg = require('fast-glob');
 
 const PROJECT_ROOT = fs.realpathSync(process.cwd());
 const DEFAULT_NAMESPACE = 'c';
 
+const paths = [];
```

Comment:
If jest is running is watch mode, caching the paths means that new modules will not be detected which might cause tests to fail. I think it worth adding a comment explaining this here.

Reply:
Comment added. I also reworked the function to call fast-glob once, which shaved a bit more time off.